models/game.rb

```diff
@@ -103,7 +103,13 @@ def ordered_players
 
   def to_h(include_actions: false, player: nil)
     actions_h = include_actions ? actions.map(&:to_h) : []
-    settings_h = include_actions ? settings.to_h : {}
+
+    settings_h =
```

Comment:
hm i wonder why we don't just always include the settings

Reply:
I'll probably have time to address comments tomorrow, want me to just make it always include settings?

Reply:
maybe

Reply:
I did this, settings is now always included, but `seed` is only included if the actions are included